packages/nuget/Sentry.AspNet/4.2.1.json

```diff
@@ -4,5 +4,30 @@
   "version": "4.2.1",
   "package_url": "https://www.nuget.org/packages/Sentry.AspNet",
   "repo_url": "https://github.com/getsentry/sentry-dotnet",
-  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/aspnet"
+  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/aspnet",
+  "categories": [
+    "server"
+  ],
+  "features": {
+    "crons": {
+      "availability": "GA",
+      "available-with-version": "4.2.0"
+    },
+    "errors": {
+      "availability": "GA",
+      "available-with-version": "1.0.0"
+    },
+    "performance": {
```

Comment:
with the many features and evolutions (span only, vitals, etc) do we need to be more granular? they key performance could be misleading?
how about tracing, transactions?